lib/mods/utils.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/**
+ * Gets the Collection of CallExpressions with callee name console.
+ *
+ * @param {any} j - the jscodemod api
+ * @param {Collection} root - the root Collection
+ * @return {Collection} - a Collection of console statements
+ *
+ * @example
+ * getConsoleStatements(j, root).filter(...);
+ */
+function getConsoleStatements(j, root) {
```

Comment:
why is this in a separate file when there is only one file that uses it? It seems like it could be in that file.